docs/spfx/use-aadhttpclient.md

```diff
@@ -170,6 +170,48 @@ If your SharePoint Framework solution requires permissions to specific resources
     > [!NOTE]
     > No matter if the administrator denies or approves the requested permissions, the solution can be deployed and used on sites. When building solutions that require additional permissions, you should never assume that the requested permissions have been granted.
 
+### Requesting permissions to Azure AD applications in another tenant
+
+The previous steps address adding a permission request to an Azure AD application defined in the same Azure AD tenant as your SharePoint Online tenant. However, when you want to grant a permission for an Azure AD application defined in another tenant (a multitenant Azure AD application), you must first add a service principal for that Azure AD application to your tenant.
+
+> [!NOTE]
+> This capability was added in the [SPFx v1.15.2 release](release-1.15.2.md).
+
+> [!IMPORTANT]
+> Azure AD applications have two types of principals. The application object is created when the Azure AD app is created; it is a *global* representation of the application across all tenants.
+>
+> Permissions are granted to service principals which are the *local* representation of the app for use in a specific tenant.
+>
+> When you create an Azure AD application in a tenant, both the application object & service principal are created in that tenant. But when you want to use the Azure AD application in another tenant, you must create a service principal for that application in your tenant.
+>
+> To learn more about Azure AD application principals, objects, and service principals, see: [Application and service principal objects in Azure Active Directory](/azure/active-directory/develop/app-objects-and-service-principals).
+
+When you want to grant SharePoint Online permission to an Azure AD application defined in another tenant, you must include two additional properties in the SPFx project's **package-solution.json** file's `webApiPermissionRequests` entry:

```

Comment:
May I suggest a different wording? "When you want to grant SharePoint Online permissions to an Azure AD application defined to another tenant, the Service Principal needs to already exist on the tenant where SharePoint Online operates. Previous to this release, SharePoint expected that Service Principal to already exist. With 1.15.2, we introduced a way to register the Service Principal during the permission request approval process. To do that, you must inclue tro additional properties in the SharePoint Frameork project's **package-solution.json** file's `webApiPermissionRequests` entry:"